core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -272,12 +296,16 @@ def validate_leading_factor_of_delay(
 class TownsFundRoundFourValidationFailure(ValidationFailure):
     """Generic Towns Fund Round 4 Validation Failure."""
 
-    tab: str
+    sheet: str
     section: str
+    column: str
     message: str
+    indexes: list[int] | int
```

Comment:
I think `row_indexes` may be clearer - what do you think?

Reply:
I prefer row indexes, we should make sure to keep this name consistent accross generic and round 4 validations

Reply:
yep agreed